docs/man/git-lfs-config.5.ronn

```diff
@@ -160,6 +160,14 @@ be scoped inside the configuration for a remote.
   not an integer, is less than one, or is not given, a value of eight will be
   used instead.
 
+* `lfs.transfer.maxretrydelay`
+
+  Specifies the maximum time in seconds LFS will wait between each retry
+  attempt. LFS uses exponential backoff for retries, doubling the time between
```

Comment:
Is this still true if the server gives us a larger value in the `Retry-After` header?

Reply:
Not quite. If I change it so that `Retry-After` counts against the retry limit, then the simplest behavior to implement is to wait for the server-provided time and then if that fails again _and_ the server does not provide another time, wait for the delay implied by the retry count. The `Retry-After` value always wins for a particular retry, but does not influence the the length of future delays.

Maybe noting that the `Retry-After` value is always used if provided but otherwise doesn't change the delay behavior in the docs is sufficient?

Reply:
I updated this paragraph to clarify the behavior when a server provides the `Retry-After` header.

Reply:
Great.  That's exactly what I was looking for.